test/integration/element_test.exs

```diff
@@ -178,4 +178,15 @@ defmodule DrabTestApp.ElementTest do
     out = find_element(:id, "inner_outer_out")
     assert visible_text(out) == "inner outer clicked"
   end
+
+
+
+  test "set_html" do
+    socket = drab_socket()
+    assert {:ok, 1} == set_html(socket, "#my_element", "<p>Hello, World!</p> ")
```

Comment:
Would be nice to check also if it was actually set, so after `set_html` check something like:

    assert %{"#my_element" => %{"innerHTML" => "<p>Hello, World!</p>"}} == query_one!(socket, "#my_element", :innerHTML}